lib/preprocess.js

```diff
@@ -8,6 +8,17 @@ import createDebug from 'debug'
 
 const debug = createDebug('spark:preprocess')
 
+export class Measurement {
+  constructor (m) {
+    for (const [key, value] of Object.entries(m)) {
+      if (['zinnia_version', 'attestation', 'participant_address'].includes(key)) continue
```

Comment:
Is it worth extracting this array into a top-level constant to save some time and memory allocations on re-creating the array for every call of the constructor?